src/createAnimatedComponent.js

```diff
@@ -37,6 +37,25 @@ function hasAnimatedNodes(value) {
   return false;
 }
 
+function flattenArray(array) {
```

Comment:
This method can be replaced with: [`array.flat(Infinity)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat)

Reply:
Which doesn't preserve `undefined` values but we don't care here.

More about this [here](https://medium.com/dailyjs/flatten-array-using-array-flat-in-javascript-ee4d0b2423e5)

Reply:
fixed 8a2e33b

Reply:
Seems like we don't support flat yet

https://stackoverflow.com/questions/52868353/javascript-array-flat-function-doesnt-define-in-react-native